style.css

```diff
@@ -152,7 +152,14 @@ button:not(:disabled):active,
 input[type="submit"]:not(:disabled):active,
 input[type="reset"]:not(:disabled):active {
   box-shadow: var(--border-sunken-outer), var(--border-sunken-inner);
-  padding: 2px 11px 0 13px;
+
+  /* Make the text to appear to shift down and right 1px without affecting any
+  surrounding layout */
+  vertical-align: -1px;
+  margin-bottom: -1px;
+  padding: 1px 0 0 2px;
```

Comment:
11px/13px to 0/2px is quite a big difference - why is that?

Reply:
Good catch -- I did not account for buttons with only a little text (where they'll be at their `min-width`) vs. a button that contains a lot of text. Fixed in 32b357b21f366240064f447b16d9e7845f374b31. Same Safari issue remains, but works as expected in Chrome and Firefox.

![CleanShot 2022-10-29 at 11 05 26](https://user-images.githubusercontent.com/3038600/198841566-6301c178-39c7-49b3-a77b-f12da96921ff.gif)